db/migrate/20171113153157_create_students.rb

```diff
@@ -0,0 +1,10 @@
+class CreateStudents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :students do |t|
+    	t.string :name
+    	t.string :email
+    	t.belongs_to :batch, index: true
+      t.timestamps
```

Comment:
Weird indentation here too

Reply:
Also, `name` and `email` should have database null-restrictions, to ensure that our database has its own protections against invalid data. 

```
  t.string :name, null: false
  t.string :email, null: false
```